ui/app/pages/send/send-content/send-amount-row/send-amount-row.component.js

```diff
@@ -26,12 +26,23 @@ export default class SendAmountRow extends Component {
     updateSendAmount: PropTypes.func,
     updateSendAmountError: PropTypes.func,
     updateGas: PropTypes.func,
+    maxMode: PropTypes.bool,
```

Comment:
Nit: we've called this `maxModeOn` elsewhere in the codebase. Admittedly the "On" part seems redundant, but it using the same name would make this value easier to search for.